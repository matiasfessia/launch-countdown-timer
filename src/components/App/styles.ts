import { css } from '@emotion/react';
import bgStars from '../../assets/images/bg-stars.svg';
import patternHills from '../../assets/images/pattern-hills.svg';

export const styles = {
  container: css`
    display: flex;
    flex: 1;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    background-color: #231d2b;
    background-image: url(${bgStars}), url(${patternHills});
    background-position: top, bottom right;
    background-repeat: no-repeat;
    margin: 0;
    height: 100vh;
  `,
  title: css`
    text-align: center;
    color: #fff;
    text-shadow: 1px 1px 3px rgb(150 150 150);
    margin: 0;
    padding: 150px 25px 50px 25px;
    font-size: 22px;
    font-weight: 300;
    letter-spacing: 5px;
    line-height: 30px;
    font-family: 'RedHatText-Regular', sans-serif;
  `,

  footer: css`
    display: flex;
    flex: 1;
    align-items: flex-end;
    justify-content: center;
    width: 100%;
    color: #fff;
  `
};
