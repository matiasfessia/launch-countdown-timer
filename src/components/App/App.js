import "./styles.css";

export const App = () => {
  return <div className="app"></div>;
};

export default App;
