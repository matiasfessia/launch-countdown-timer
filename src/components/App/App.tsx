/** @jsxImportSource @emotion/react */
import { Global } from '@emotion/react';
import { SocialMedia } from '../SocialMedia';

import { Timer } from '../Timer';
import { styles } from './styles';

export const App = () => {
  return (
    <div css={styles.container}>
      <Global styles={styles} />
      <h1 css={styles.title}>WEÂ´RE LAUNCHING SOON</h1>
      <Timer />
      <div css={styles.footer}>
        <SocialMedia />
      </div>
    </div>
  );
};

export default App;
