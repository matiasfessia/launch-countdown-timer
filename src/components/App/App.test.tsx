import { render } from '@testing-library/react';
import App from './App';

it('renders correctly', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
