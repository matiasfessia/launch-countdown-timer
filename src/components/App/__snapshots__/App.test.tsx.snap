// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="css-g2gst9-App"
  >
    <h1
      class="css-1eu2r4m-App"
    >
      WEÂ´RE LAUNCHING SOON
    </h1>
    <div
      class="css-1xpqvqu-Timer"
    >
      <div
        class="css-1822xkh-FlipCard"
      >
        <div
          class="css-y5ro21-FlipCard"
          data-value="09"
        >
          <div
            class="css-nq5f8h-FlipCard"
            data-value="09"
          />
          <div
            class="css-xp1t8d-FlipCard"
          />
          <div
            class="css-18hlcob-FlipCard"
          />
        </div>
        <h2
          class="css-xzkfzk-FlipCard"
        >
          days
        </h2>
      </div>
      <div
        class="css-1822xkh-FlipCard"
      >
        <div
          class="css-y5ro21-FlipCard"
          data-value="12"
        >
          <div
            class="css-nq5f8h-FlipCard"
            data-value="12"
          />
          <div
            class="css-xp1t8d-FlipCard"
          />
          <div
            class="css-18hlcob-FlipCard"
          />
        </div>
        <h2
          class="css-xzkfzk-FlipCard"
        >
          hours
        </h2>
      </div>
      <div
        class="css-1822xkh-FlipCard"
      >
        <div
          class="css-y5ro21-FlipCard"
          data-value="41"
        >
          <div
            class="css-nq5f8h-FlipCard"
            data-value="41"
          />
          <div
            class="css-xp1t8d-FlipCard"
          />
          <div
            class="css-18hlcob-FlipCard"
          />
        </div>
        <h2
          class="css-xzkfzk-FlipCard"
        >
          minutes
        </h2>
      </div>
      <div
        class="css-1822xkh-FlipCard"
      >
        <div
          class="css-y5ro21-FlipCard"
          data-value="56"
        >
          <div
            class="css-nq5f8h-FlipCard"
            data-value="56"
          />
          <div
            class="css-xp1t8d-FlipCard"
          />
          <div
            class="css-18hlcob-FlipCard"
          />
        </div>
        <h2
          class="css-xzkfzk-FlipCard"
        >
          seconds
        </h2>
      </div>
    </div>
    <div
      class="css-7exyml-App"
    >
      <div
        class="css-20wlgm-SocialMedia"
      >
        <a
          class="css-1eiqcch-SocialMedia"
          href="https://www.facebook.com/"
        >
          <img
            alt="facebook icon"
            class="css-1idlvul-SocialMedia"
            src="icon-facebook.svg"
          />
        </a>
        <a
          class="css-1eiqcch-SocialMedia"
          href="https://www.pinterest.com/"
        >
          <img
            alt="pinterest icon"
            class="css-1idlvul-SocialMedia"
            src="icon-pinterest.svg"
          />
        </a>
        <a
          class="css-1eiqcch-SocialMedia"
          href="http://instagram.com/"
        >
          <img
            alt="instagram icon"
            class="css-1idlvul-SocialMedia"
            src="icon-instagram.svg"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;
